test/FunctionalTests/Microsoft.OData.Edm.Tests/Validation/EdmErrorTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmErrorTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using Microsoft.OData.Edm.Validation;
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.Validation
+{
+    public class EdmErrorTests
+    {
+        private EdmLocation errorLocation;
+        private EdmErrorCode errorCode;
+        private string errorMessage;
+        private EdmErrorLevel errorLevel;
+
+        public EdmErrorTests()
+        {
+            errorCode = EdmErrorCode.EnumMemberMustHaveValue;
+            errorMessage = "Enum Member Must Have Value";
+            errorLocation = null;
+            errorLevel = EdmErrorLevel.Error;
+        }
+
+        [Fact]
+        public void EdmErrorConstructorWithNoEdmErrorLevelSHouldSetEdmErrorLevelToNone()
+        {
+            string expectedString = "EnumMemberMustHaveValue : Enum Member Must Have Value";
+            var edmError = new EdmError(errorLocation, errorCode, errorMessage);
+            Assert.Equal(errorCode, edmError.ErrorCode);
+            Assert.Equal(errorMessage, edmError.ErrorMessage);
+            Assert.Equal(EdmErrorLevel.None, edmError.ErrorLevel);
+            Assert.Equal(expectedString, edmError.ToString());
+        }
+
+        [Fact]
+        public void EdmErrorConstructorWithEdmErrorLevelSHouldSetEdmErrorLevelCorrectly()
+        {
+            string expectedString = "EnumMemberMustHaveValue : Enum Member Must Have Value : Error";
+            var edmError = new EdmError(errorLocation, errorCode, errorMessage, errorLevel);
+            Assert.Equal(errorCode, edmError.ErrorCode);
+            Assert.Equal(errorMessage, edmError.ErrorMessage);
+            Assert.Equal(errorLevel, edmError.ErrorLevel);
+            Assert.Equal(expectedString, edmError.ToString());
+        }

```

Comment:
Add test cases to cover the ToString() changes

Reply:
Resolved.